pod.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
what's that ?

Reply:
oh accidentally created that, for testing purposes, removing it now.